compiler/next/include/chpl/resolution/resolution-types.h

```diff
@@ -137,6 +137,20 @@ class UntypedFnSignature {
   bool operator!=(const UntypedFnSignature& other) const {
     return !(*this == other);
   }
+  static bool update(owned<UntypedFnSignature>& keep,
+                     owned<UntypedFnSignature>& addin) {
+    return defaultUpdateOwned(keep, addin);
+  }
+  void mark(Context* context) const {
+    id_.mark(context);
+    name_.mark(context);
+    for (auto const& elt : formals_) {
+      elt.name.mark(context);
+      context->markUnownedPointer(elt.decl);
+    }
+    context->markUnownedPointer(whereClause_);
+  }
```

Comment:
Interesting case for the linter. We'd have to check collection members and that each field of the collection T is mentioned

Reply:
IMO it'd be enough to just check all fields are mentioned.